docs/source/API/core/policies/RangePolicy.rst

```diff
@@ -11,10 +11,14 @@ Usage
 
 .. code-block:: cppkokkos
 
-    Kokkos::RangePolicy<>(begin, end, args...)
-    Kokkos::RangePolicy<ARGS>(begin, end, args...)
-    Kokkos::RangePolicy<>(Space(), begin, end, args...)
-    Kokkos::RangePolicy<ARGS>(Space(), begin, end, args...)
+    Kokkos::RangePolicy<>(begin, end)
+    Kokkos::RangePolicy<ARGS>(begin, end)
+    Kokkos::RangePolicy<>(begin, end, chunk_size)
+    Kokkos::RangePolicy<ARGS>(begin, end, chunk_size)
+    Kokkos::RangePolicy<>(Space(), begin, end)
+    Kokkos::RangePolicy<ARGS>(Space(), begin, end)
+    Kokkos::RangePolicy<>(Space(), begin, end, chunk_size)
+    Kokkos::RangePolicy<ARGS>(Space(), begin, end, chunk_size)
```

Comment:
```suggestion
    Kokkos::RangePolicy<>(begin, end)
    Kokkos::RangePolicy<>(begin, end, chunk_size)
    Kokkos::RangePolicy<>(Space(), begin, end)
    Kokkos::RangePolicy<>(Space(), begin, end, chunk_size)
```
What would `ARGS` be?

Reply:
That was the original documentation.  I'll change it to `...` in all places.